client/state/sites/selectors/get-site-collisions.js

```diff
@@ -14,21 +12,18 @@ import getSitesItems from 'calypso/state/selectors/get-sites-items';
  * @param  {object}   state Global state tree
  * @returns {number[]}       WordPress.com site IDs with collisions
  */
-export default createSelector(
-	( state ) =>
-		map(
-			filter( getSitesItems( state ), ( wpcomSite ) => {
-				const wpcomSiteUrlSansProtocol = withoutHttp( wpcomSite.URL );
-				return (
-					! wpcomSite.jetpack &&
-					some(
-						getSitesItems( state ),
-						( jetpackSite ) =>
-							jetpackSite.jetpack && wpcomSiteUrlSansProtocol === withoutHttp( jetpackSite.URL )
-					)
-				);
-			} ),
-			'ID'
-		),
-	getSitesItems
-);
+export default createSelector( ( state ) => {
+	const sitesItems = Object.values( getSitesItems( state ) );
+	return sitesItems
+		.filter( ( site ) => {
+			const siteUrlSansProtocol = withoutHttp( site.URL );
+			return (
+				! site.jetpack &&
+				sitesItems.some(
+					( jetpackSite ) =>
+						jetpackSite.jetpack && siteUrlSansProtocol === withoutHttp( jetpackSite.URL )
+				)
```

Comment:
Do we really want to execute this for all `sitesItems`? I think we can move this to a separate constant above and reuse it here.

Reply:
What exactly could be moved to a constant? I don't see any opportunity to do that. `siteUrlSansProtocol` is different for each iteration.

Reply:
Ha, you're right, that was an oversight. Disregard.